### PR TITLE
Feature/change blog owner improved

### PR DESCRIPTION
**Pull request by vladnicoara**

### CODE REVIEW

File: client/app/scripts/liveblog-edit/module.js [hunk 1]
```diff
@@ -232,8 +231,8 @@ define([
             },
             buildOwner: function(userID) {
                 api('users').getById(userID).then(function(data) {
-                    vm.original_creator = data;
-                    vm.lastOwnerId = userID;
+                    vm.temp_chosen = vm.original_creator = data;
```
Comment (vied12) [inline client/app/scripts/liveblog-edit/module.js]: Do you really need `temp_chosen` ?
If yes, can you explain here with a comment in the code his role and rename it with a meaningful name please ? :)

File: client/app/scripts/liveblog-edit/views/settings.html [hunk 2]
```diff
@@ -25,8 +25,8 @@ <h3 class="page-nav-title">Settings</h3>
                 <li ng-class="{active: settings.generalTab}">
                     <a ng-href="" ng-click="settings.generalTab = true">General</a>
                 </li>
-                <li ng-class="{active: !settings.generalTab}" data="blog-settings-users">
-                    <a ng-href="" ng-click="settings.generalTab = false">Users</a>
+                <li ng-class="{active: !settings.generalTab}" data="blog-settings-team">
+                    <a ng-href="" ng-click="settings.generalTab = false">Team</a>
```
Comment (nistormihai) [inline client/app/scripts/liveblog-edit/views/settings.html]: translate attribute is missing

File: client/app/scripts/liveblog-edit/views/settings.html [hunk 3]
```diff
@@ -69,41 +69,63 @@ <h2 translate>General</h2>
...
+                                                <img data-src="settings.original_creator.user.picture_url">
+                                            </figure>
+                                        </div>
+                                    </div>
+                                    <div class="inline-content">
+                                        <div class="form-input" data="original-creator-display-name">
+                                           <strong>{{ settings.original_creator.display_name }}</strong>
+                                        </div>
+                                        <div class="form-input" data="original-creator-username">
+                                            {{ settings.original_creator.username }}
```
Comment (plamut) [inline client/app/scripts/liveblog-edit/views/settings.html]: Does the original creator change? If not, one-time binding can be used here (and the same for the display_name a few lines above).

File: client/app/scripts/liveblog-edit/views/settings.html [hunk 4]
```diff
@@ -69,41 +69,63 @@ <h2 translate>General</h2>
...
+                                                <div>
+                                                    <form>
+                                                        <label for="assignee-search" translate>Chose new owner</label>
+                                                        <input type="text" name="assignee-search" ng-model="settings.search">
+                                                    </form>
+                                                    <div class="search-result">
+                                                        <ul class="users-list-embed">
+                                                            <li ng-repeat="user in settings.avUsers | filter:settings.search" ng-click="settings.temp_selected_owner = user" ng-class="{'active': settings.temp_selected_owner._id === user._id}">
+                                                                <figure class="avatar small"><img data-src="user.picture_url"></figure>
+                                                                <div class="name">{{user.display_name}}</div>
```
Comment (plamut) [inline client/app/scripts/liveblog-edit/views/settings.html]: Add one-time binding? User's display_name probably doesn't change?

Comment (nistormihai) [inline client/app/scripts/liveblog-edit/views/settings.html]: you can change the owner, so yeah one-time binding doesn't work here.

Comment (plamut) [inline client/app/scripts/liveblog-edit/views/settings.html]: OK, just wanted to make sure.

**Comment by nistormihai** [comment]
in rest LGTM

**Comment by vied12** [comment]
LGTM but needs a rebase

**Pull request closed without merging**